ninja.lua

```diff
@@ -135,6 +135,10 @@ function ninja.shesc(value)
 	return value
 end
 
+function ninja.supportsprj(prj)
```

Comment:
I dislike that name which conflict with action supports which take the list provided by ninja...
How about `can_generate`?